QgisModelBaker/gui/validate.py

```diff
@@ -632,12 +650,50 @@ def _auto_zoom_button_clicked(self):
         if self.auto_zoom_button.isChecked:
             self.auto_pan_button.setChecked(False)
 
-    def _select_validator_config_file(self):
+    def _select_config_file(self):
         filename, _ = QFileDialog.getOpenFileName(
             self, self.tr("Select the validator config file")
         )
         if filename:
-            self.validator_config_file_line_edit.setText(filename)
+            self.config_file_line_edit.setText(self._relative_path(filename))
+
+    def _save_config_file(self):
```

Comment:
I would name it `_save_config_file_path` to make it clear that the path to the file is stored, and not the entire file itself.

Maybe in the gui as well.